cactus_progressive_config.xml

```diff
@@ -54,11 +85,13 @@
 		minimumDistance="0.01" 
 		minimumSequenceLengthForBlast="30"
 		annealingRounds="128" 
-		deannealingRounds="2 8" 
+		deannealingRounds="2 8"
 		blockTrim="5" 
 		minimumTreeCoverage="0.0" 
 		trim="0 0" 
 		minimumBlockDegree="2"
+                autoMinimumIngroupDegree="0"
```

Comment:
Can get rid of this, it was an attempt to only enforce minimumIngroupDegree=2 on the nodes that need it the most. But setting minimumIngroupDegree=1 without removing recoverable chains causes coverage loss on almost *every* node, no matter the divergence.